features.html

```diff
@@ -184,39 +224,53 @@
 
     <div class="feature-group grid">
       <div class="feature">
+        <div id="contract-module" class="feature-anchor"></div>
+        <div class="feature-link"><a class="icon-link" href="#contract-module"></a></div>
         {% include feature-gfy.html n="PessimisticFlashyAustraliancurlew" %}
         <p><b>Contract module</b>: the opposite of <i>Expand Module</i> refactoring. Available only if there are no
           child modules.</p>
       </div>
 
       <div class="feature">
+        <div id="create-module" class="feature-anchor"></div>
+        <div class="feature-link"><a class="icon-link" href="#create-module"></a></div>
         {% include feature-gfy.html n="HauntingAgonizingDuck" %}
         <p><b>Create module file</b>: if you have <code>mod foo;</code>, but no <code>foo.rs</code>
           this quick fix will create the missing file.</p>
       </div>
 
       <div class="feature">
+        <div id="extract-module" class="feature-anchor"></div>
+        <div class="feature-link"><a class="icon-link" href="#extract-module"></a></div>
         {% include feature-gfy.html n="BleakWholeGalapagospenguin" %}
         <p><b>Extract inline module</b>: move <code>mod foo { ... }</code> to a separate file.</p>
       </div>
 
-      <div class="feature" id="add-derive">
+      <div class="feature">
+        <div id="derive-clause" class="feature-anchor"></div>
+        <div class="feature-link"><a class="icon-link" href="#derive-clause"></a></div>
```

Comment:
I'd prefer to stick with with existing ids (so `#derive-clause` -> `#add-derive`). @matklad posted links pointing here somewhere and it would be good not to break them.